snippets/java.json

```diff
@@ -26,24 +25,15 @@
             "import javax.persistence.GeneratedValue;",
             "import javax.persistence.GenerationType;",
             "import javax.persistence.Id;",
+            "import lombok.Data;",
             "",
+            "@Data",
```

Comment:
Adding lombok data. To use this you should add this dependency,e.g. in pom.xml file:
		```<dependency>
			<groupId>org.projectlombok</groupId>
			<artifactId>lombok</artifactId>
			<optional>true</optional>
		</dependency>```

Reply:
Hi Adborah!  Thanks for the feedback.  This is a reasonable request, but I would prefer to keep a "vanilla java" approach for those who do not use Lombok.  I personally like Lombok though so may add a new lombok specific snippet with your changes.